api/envoy/config/route/v3/route_components.proto

```diff
@@ -619,7 +629,7 @@ message RouteMatch {
   // match. The router will check the query string from the ``path`` header
   // against all the specified query parameters. If the number of specified
   // query parameters is nonzero, they all must match the ``path`` header's
-  // query string for a match to occur.  In the event query parameters are
+  // query string for a match to occur. In the event query parameters are
```

Comment:
This change is unrelated, but without it I got an error message from the pre-push check:

```
ERROR: From ./api/envoy/config/route/v3/route_components.proto
ERROR: ./api/envoy/config/route/v3/route_components.proto:632: over-enthusiastic spaces
ERROR: check format failed. run 'bazel run //tools/code_format:check_format -- fix'
```

Please let me know if I should revert it. Thanks!